test/system/500-networking.bats

```diff
@@ -754,4 +754,22 @@ EOF
     done
 }
 
+@test "podman rootless netns work with symlink" {
+    is_rootless || skip "only meaningful for rootless"
+    if ! readlink /var/run; then
+        skip "/var/run is not a symlink: cannot test this bug"
+    fi
+    NEW_XDG_RUNTIME_DIR=$XDG_RUNTIME_DIR
+    if [[ "${XDG_RUNTIME_DIR:0:8}" != "/var/run" ]]; then
+        if [[ "${XDG_RUNTIME_DIR:0:4}" != "/run" ]]; then
+            skip "XDG_RUNTIME_DIR: \"$XDG_RUNTIME_DIR\" does not point to /run/...: cannot test this bug"
+        fi
+        NEW_XDG_RUNTIME_DIR="/var$XDG_RUNTIME_DIR"
+    fi
+
+    # NOTE: the --root/--runroot are required to force a new boltdb and not reuse old chached paths
+    XDG_RUNTIME_DIR="$NEW_XDG_RUNTIME_DIR" run_podman --root $PODMAN_TMPDIR/root --runroot $PODMAN_TMPDIR/runroot unshare --rootless-netns ip a
+    is "$output" ".*tap0.*" "slirp4netns interface exists in netns"
```

Comment:
I just ran this against podman @ fe8e53632 (main) and it passes. This suggests that you are not testing what you think you're testing.

Reply:
Should fail with main now and pass on this PR if netavark is used.